go/chat/tlf.go

```diff
@@ -176,23 +186,27 @@ func (t *KBFSNameInfoSource) PublicCanonicalTLFNameAndID(ctx context.Context, tl
 	group, ectx := errgroup.WithContext(BackgroundContext(ctx, t.G()))
 
 	var ib []keybase1.TLFIdentifyFailure
+	doneCh := make(chan struct{})
 	if identBehavior != keybase1.TLFIdentifyBehavior_CHAT_SKIP {
 		group.Go(func() error {
 			var err error
 			names := utils.SplitTLFName(tlfName)
-			ib, err = t.Identify(ectx, names, false, identBehavior)
+			ib, err = t.Identify(ectx, names, false,
```

Comment:
this `ib`, too...i don't think anything looks at it?

Reply:
Oops, that's a bug, good catch!